sktime/registry/_base_classes.py

```diff
@@ -77,6 +81,16 @@
     ("regressor", BaseRegressor, "time series regressor"),
     ("forecaster", BaseForecaster, "forecaster"),
     ("transformer", BaseTransformer, "time series transformer"),
+    (
+        "transformer-pairwise-tab",
+        BasePairwiseTransformer,
+        "pairwise transformer for tabular data, distance or kernel",
+    ),
+    (
+        "transformer-pairwise-ts",
```

Comment:
Why is this 'transformer-pairwise-ts'. Wouldn't something like 'transformer-pairwise-pan' be more inline with the other you've named 'transformer-pairwise-tab'

Reply:
fixed